.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "third-party/ios-cmake"]
 	path = third-party/ios-cmake
-	url = git@github.com:aukilabs/ios-cmake.git
+	url = https://github.com/aukilabs/ios-cmake.git
```

Comment:
which branch you want for this repo? I dont see your commit for it, why did we fork it?

Reply:
its `release/v4.5.0` as specified below
i didn't need to push any fixes to it yet, i recall they had a bug with visionOS but i haven't used that yet